pkg/splunk/client/names.go

```diff
@@ -0,0 +1,31 @@
+package client
+
+const (
+	// Azure token fetch URL
+	azureTokenFetchUrl = "http://169.254.169.254/metadata/identity/oauth2/token"
+
+	// Azure http header XMS version
+	azureHttpHeaderXmsVersion = "2017-11-09"
```

Comment:
I retract my previous comment if this is the version we're going with.

Reply:
yes that's correct